src/solvers/mp/rcspp/PrincipalGraph.h

```diff
@@ -0,0 +1,56 @@
+//
+// Created by antoine legrain on 2020-04-11.
+//
+
+#ifndef NURSESCHEDULER_PRINCIPALGRAPH_H
+#define NURSESCHEDULER_PRINCIPALGRAPH_H
+
+#include "tools/MyTools.h"
+#include "RCGraph.h"
+
+class SubProblem;
+
+class PrincipalGraph {
```

Comment:
Add comment to describe class?

Reply:
done in  6efc95cf3ddeb8d97e69f945b603238e901ba853